src/views/blocks/tickets/quantity-remove.php

```diff
@@ -13,20 +13,20 @@
  * @since 4.9.3
  * @since 4.11.3 Updated the button to include a type - helps avoid submitting forms unintentionally.
  * @since 4.11.4 Added accessibility classes to screen reader text element.
+ * @since TBD    Removed `type="submit"` from button element, as it's its default.
  *
- * @version 4.11.4
+ * @version TBD
  *
  * @var $this Tribe__Tickets__Editor__Template
  */
 
 $ticket = $this->get( 'ticket' );
 $button_title = sprintf(
-	_x('Decrease ticket quantity for %s', '%s: ticket name.', 'event-tickets'),
+	_x( 'Decrease ticket quantity for %s', '%s: ticket name.', 'event-tickets' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 8eb0acd7a3a98a976681aa50c87fca9b</sub>

Reply:
```suggestion
	/// translators: %s: ticket name.
	_x( 'Decrease ticket quantity for %s', 'Decrease ticket quantity button title', 'event-tickets' ),
```